modules/registry-operator-config-resources-secret.adoc

```diff
@@ -45,12 +49,20 @@ $ oc create secret generic image-registry-private-configuration-user --from-lite
 
 *For GCP:*
 
-* Create an {product-title} Secret that contains the required keys.
+* Create an {product-title} Secret that contains the required key.
 +
 ----
 $ oc create secret generic image-registry-private-configuration-user --from-file=REGISTRY_STORAGE_GCS_KEYFILE=<path_to_keyfile> --namespace openshift-image-registry
 ----
 
+*For Azure:*
+
+* Create an {product-title} Secret that contains the required key.
++
+----
+$ oc create secret generic image-registry-private-configuration-user --from-file=REGISTRY_STORAGE_AZURE_ACCOUNTKEY=<path_to_accountkey> --namespace openshift-image-registry
```

Comment:
An Azure access key is a string that you can copy from the Azure web console, so I think it's better to use `--from-literal=REGISTRY_STORAGE_AZURE_ACCOUNTKEY=<accountkey>`